src/main/java/spoon/support/visitor/java/JavaReflectionTreeBuilder.java

```diff
@@ -296,7 +296,7 @@ public void visitField(Field field) {
 				CtLiteral<Object> defaultExpression = factory.createLiteral(field.get(null));
 				ctField.setDefaultExpression(defaultExpression);
 			}
-		} catch (IllegalAccessException e) {
```

Comment:
What about catching "IllegalAccessException | ExceptionInInitializerError"?

Thanks a lot for the changes!